docs/apps/julia.md

```diff
@@ -0,0 +1,103 @@
+# Julia
+
+## Description
+
+Julia is a high-level, high-performance dynamic programming language for
+numerical computing. It provides a sophisticated compiler, distributed
+parallel execution, numerical accuracy, and an extensive mathematical
+function library.
+
+For a quick introduction and tutorial, see https://github.com/csc-training/julia-introduction 
+
+[TOC]
+
+## Available
+
+### Puhti
+-   1.1.1 compiled with Intel Math Kernel Library (MKL)
```

Comment:
The file appl/soft/math/julia/gcc-mkl/julia-1.1.1/usr has the wrong permissions, and is not accessible